c7n/filters/iamaccess.py

```diff
@@ -211,9 +211,17 @@ def normalize_conditions(self, s):
     # Condition handlers
 
     # kms specific
+    # https://docs.aws.amazon.com/kms/latest/developerguide/policy-conditions.html#conditions-kms
     def handle_kms_calleraccount(self, s, c):
+        raise Exception(self.allowed_accounts)
```

Comment:
this doesnt look right, why raise here unconditionally?

Reply:
I was debugging and forgot about this, thanks for the catch. Fixed.